modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java

```diff
@@ -1643,25 +1650,27 @@ private void _populateDDMDataProviderNavigationItem(
 	private List<DDMFormFieldType> _removeDDMFormFieldTypesOutOfScope(
 		List<DDMFormFieldType> ddmFormFieldTypes) {
 
-		List<DDMFormFieldType> availableDDMFormFieldTypes = new ArrayList<>();
+		List<String> ddmFormFieldTypesOutofScope = Arrays.asList(
```

Comment:
Could you please change `ddmFormFieldTypesOutofScope` to `ddmFormFieldTypesOutOfScope`?